.github/workflows/format.yml

```diff
@@ -0,0 +1,39 @@
+name: 'Format Code'
+
+on:
+  push:
+    branches:
+      - main
+
```

Comment:
dose this action need any specific permissions?

see the following for example:
https://github.com/astrolicious/studiocms/blob/7092e69d0eb1e4a4477444532117af9eab9736ed/.github/workflows/changeset-release.yml#L11-L24

Reply:
probably